packages/manager/modules/pci/src/projects/new/payment/components/add/index.html

```diff
@@ -0,0 +1,48 @@
+<div class="pci-project-new-payment-method-add">
+    <div
+        class="row"
+        data-ng-repeat="authorizedPaymentMethods in $ctrl.authorizedPaymentMethods.chunks track by $index"
```

Comment:
The `col-md-6` isn't enough to have two elements per `row` ?

Reply:
I'm not sure to understand: you mean that there is no need to have this `ng-repeat` to create rows for the cols?
I think that a `div.row` parent is required for `div.col-*`

Reply:
What I meant is that it might not be needed to split the initial array into chunks of 2 arrays each